packages/@aws-cdk/tmp-toolkit-helpers/src/api/io/toolkit-action.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * The current action being performed by the CLI. 'none' represents the absence of an action.
+ */
+export type ToolkitAction =
```

Comment:
Unrelated to this PR but it just caught my so putting it out here. This list is slightly different than then one in the CLI. Intentional? 

https://github.com/aws/aws-cdk-cli/blob/01acccf590c67d10360ac46990958252a0663e44/packages/aws-cdk/lib/toolkit/cli-io-host.ts#L91-L110

Reply:
Yes, the CLI has more. To be unified soon.